juju/charmhub.py

```diff
@@ -0,0 +1,24 @@
+from .client import client
+from .errors import JujuError
+
+
+class CharmHub:
+    def __init__(self, model):
+        self.model = model
+
+    async def info(self, name, channel=None):
+        """info displays detailed information about a CharmHub charm. The charm
+        can be specified by name or by path.
```

Comment:
What does path mean in this context? The first thing I think of is a local charm

Reply:
I robbed it from Juju, I'll fix